dmoj/settings.py

```diff
@@ -11,152 +11,159 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 import datetime
 import os
+from typing import Any, Dict, List, Literal, Optional, Sequence, Set, TYPE_CHECKING, Tuple, Union
 
 from django.utils.translation import gettext_lazy as _
 from django_jinja.builtins import DEFAULT_EXTENSIONS
 from jinja2 import select_autoescape
 
+import django_stubs_ext  # noqa: I100, I102, I202
+django_stubs_ext.monkeypatch()
```

Comment:
is it necessary to keep `monkeypatch()` out of `TYPE_CHECKING`?

Reply:
Yeah, we need it when running in production as well. Otherwise, we still get the same `TypeError` (ref: https://github.com/typeddjango/django-stubs#i-cannot-use-queryset-or-manager-with-type-annotations)